ncm-grub/src/test/resources/md5password.pan

```diff
@@ -0,0 +1,15 @@
+object template md5password;
+
+include 'base';
+
+
```

Comment:
include "components/grub/config";
here would make the travis panlint check pass.

although it's not actually required by pan due to its inclusion via base. But I guess we live by the rules of panlint @stdweird @jrha ?

Reply:
i will happily merge this without extra include. panlint on tests is interpreted a bit less strict also by @jrha.